src/parser/markdown.ts

```diff
@@ -54,6 +54,9 @@ mdit.use(attributes);
 
 // MARK: untyped plugins done
 
+import githubAlerts from 'markdown-it-github-alerts';
+mdit.use(githubAlerts);
```

Comment:
@jannis-baum

All other plugins have this `/* @ts-expect-error: markdown-it modules aren't typed */` before the imports, but this one would error out of build with that.

Is this really the first "typed" plugin?

Reply:
The one you are using is one of the few that are actually typed ^^ You can tell by the blue "TS" logo next to the heading: https://www.npmjs.com/package/markdown-it-github-alerts

Maybe this means we should adjust the comments to say "most ...". The graphviz and anchor plugins in the bottom of that file are also typed.

Reply:
Oh yes, the "markdown-it modules aren't typed" is just a comment and not a string match to the error? :smile:

Reply:
Ok I can move GHAlert import to the top like done with the other ones and adjust the comments as well

Reply:
> Oh yes, the "markdown-it modules aren't typed" is just a comment and not a string match to an error message? 😄

It looks like a comment but it's actually a directive for the TS compiler. `@ts-expect-error` tells it to ignore the error that will happen for lack of types in the next line and just keep the code as it is. The part after that is a mandatory description of the error that I wrote because it will complain and still fail if there isn't one ^^

Reply:
Lol what a pain

I think since you have the full info at the top comment, they can just say "module not typed" or w/e?